modules/crypt.cpp

```diff
@@ -501,25 +504,36 @@ class CCryptMod : public CModule {
         }
     }
 
+    void OnNickPrefixCommand(const CString& sCommand) {
+        CString sPrefix = sCommand.Token(1, true);
+
+        if (sPrefix.empty()) {
+            PutModule("Nick Prefix: " + NickPrefix());
+        } else {
+            CString sStatusPrefix = GetUser()->GetStatusPrefix();
+            size_t sp = sStatusPrefix.size();
+            size_t np = sPrefix.size();
+            SetNV(NICK_PREFIX_KEY, sPrefix);
+            PutModule("Setting Nick Prefix to " + sPrefix);
+            if (sStatusPrefix.CaseCmp(sPrefix, std::min(sp, np)) == 0)
+                PutModule("WARNING: overlap with Status Prefix (" + sStatusPrefix + "), this Nick Prefix will not be used!");
```

Comment:
Either "*" or ".", see NickPrefix(). Thought I'd allow setting it in case one changes it's status prefix instead afterwards.